app/js/components/pages/Job.jsx

```diff
@@ -115,7 +115,14 @@ const Job = ({ action, inputs, metadata, submitJob, cancelJob, children }) => {
                                 }
                             </select>
                         )
-                        : type === 'MetadataCategory' ?
+                        /* TODO how should unsupported MetadataColumn type
```

Comment:
Note: the current implementation allows this page to render any type expressions that start with `'MetadataColumn'` in the same manner (i.e. metadata file dropdown + column name text field). If the type expression isn't actually supported, once the user hits "Go", a `NotImplementedError` will be raised on the Python side and displayed to the user. Perhaps this behavior is good enough (for now)? Open to suggestions for how to handle this more elegantly.

Reply:
This sounds good enough ™️ to me, thanks for adding the TODO note for future maintainers!